assets/scripts/foreground.js

```diff
@@ -1,77 +1,78 @@
-  jQuery('li#t-permalink a').prepend('<div class="drop-icon"><i class="fa fa-dot-circle-o fa-fw"></i></div>')
-  jQuery('li#t-info a').prepend('<div class="drop-icon"><i class="fa fa-info fa-fw"></i></div>')
-  jQuery('li#feedlinks a').prepend('<div class="drop-icon"><i class="fa fa-rss fa-fw"></i></div>')
-  
-  jQuery('ul#toolbox-dropdown.dropdown>li#n-recentchanges a').prepend('<div class="drop-icon"><i class="fa fa-tasks fa-fw"></i></div>')
-  jQuery('ul#toolbox-dropdown.dropdown>li#n-help a').prepend('<div class="drop-icon"><i class="fa fa-question fa-fw"></i></div>')
+	jQuery( 'li#t-smwbrowselink a' ).prepend( '<div class="drop-icon"><i class="fa fa-eye fa-fw"></i></div>' );
+	jQuery( 'li#t-whatlinkshere a' ).prepend( '<div class="drop-icon"><i class="fa fa-arrows fa-fw"></i></div>' );
+	jQuery( 'li#t-blockip a' ).prepend( '<div class="drop-icon"><i class="fa fa-ban fa-fw"></i></div>' );
+	jQuery( 'li#t-recentchangeslinked a' ).prepend( '<div class="drop-icon"><i class="fa fa-bars fa-fw"></i></div>' );
+	jQuery( 'li#t-contributions a' ).prepend( '<div class="drop-icon"><i class="fa fa-smile-o fa-fw"></i></div>' );
+	jQuery( 'li#t-log a' ).prepend( '<div class="drop-icon"><i class="fa fa-bars fa-fw"></i></div>' );
+	jQuery( 'li#t-emailuser a' ).prepend( '<div class="drop-icon"><i class="fa fa-envelope fa-fw"></i></div>' );
+	jQuery( 'li#t-userrights a' ).prepend( '<div class="drop-icon"><i class="fa fa-gavel fa-fw"></i></div>' );
+	jQuery( 'li#t-upload a' ).prepend( '<div class="drop-icon"><i class="fa fa-upload fa-fw"></i></div>' );
+	jQuery( 'li#t-specialpages a' ).prepend( '<div class="drop-icon"><i class="fa fa-magic fa-fw"></i></div>' );
+	jQuery( 'li#t-print a' ).prepend( '<div class="drop-icon"><i class="fa fa-print fa-fw"></i></div>' );
+	jQuery( 'li#t-permalink a' ).prepend( '<div class="drop-icon"><i class="fa fa-dot-circle-o fa-fw"></i></div>' );
+	jQuery( 'li#t-info a' ).prepend( '<div class="drop-icon"><i class="fa fa-info fa-fw"></i></div>' );
+	jQuery( 'li#feedlinks a' ).prepend( '<div class="drop-icon"><i class="fa fa-rss fa-fw"></i></div>' );
 
+	jQuery( 'ul#toolbox-dropdown.dropdown>li#n-recentchanges a' ).prepend( '<div class="drop-icon"><i class="fa fa-tasks fa-fw"></i></div>' );
+	jQuery( 'ul#toolbox-dropdown.dropdown>li#n-help a' ).prepend( '<div class="drop-icon"><i class="fa fa-question fa-fw"></i></div>' );
 
-  // Turn categories into labels
-  jQuery('#mw-normal-catlinks ul li a').addClass('label');
+	// Turn categories into labels
+	jQuery( '#mw-normal-catlinks ul li a' ).addClass( 'label' );
 
-  // Make the Page Action button respond to hover
-  jQuery('a.button.dropdown').mouseenter(function(){
-    jQuery('ul#drop1').addClass('open').addClass('right').css('top', '32px').css('left', '785px');
-  });
-  jQuery('ul#drop1').mouseleave(function(){
-    jQuery('ul#drop1').removeClass('open').css('top', '-9999px').css('left', '785px');
-  });
+	// Make the Page Action button respond to hover
```

Comment:
If you use native Foundation on hover, this will cause issues and breaks the on hover of the action button.